### ISSUE TITLE
Feature request: add plugin badges

### ISSUE THREAD

**Issue by binarynoise** [BetterDiscord/docs#32]
@Huderon created some great badges on Discord.
Could you please add it somewhere so we can use it in our READMEs, like https://docs.betterdiscord.app/branding/ which appears to be https://github.com/BetterDiscord/docs/tree/main/docs/public/branding ?

**Comment by zerebos** [comment]
I think I'd be more apt to add it to the actual website rather than just the docs. But I'm open to other thoughts here.

**Comment by binarynoise** [comment]
For me as plugin dev it wouldn't make much of a difference.
I only asked here because I found this repo and it contained the other logos.

**Comment by binarynoise** [comment]
Is there a repo for the main website somewhere? I'd create a PR there

**Comment by Huderon** [comment]
Nah, unfortunately none of that is public. I'd honestly just attach it directly to your readme for now. There's no real time frame for anything to do with the site, it's a bit of a mess to get anything done with it from what I've heard

**Comment by zerebos** [comment]
@Huderon can you post the badge here so I can add it to the site?

**Comment by Huderon** [comment]
![Image](https://github.com/user-attachments/assets/b63e35fe-4789-44e5-a7f8-8eeb783d0e4a)

![Image](https://github.com/user-attachments/assets/a496edb3-e9dd-43a4-ab2b-983791adb476)

![Image](https://github.com/user-attachments/assets/0a85ed6f-cf48-438e-a90b-b78a580aad55)

![Image](https://github.com/user-attachments/assets/e35bab10-5892-403a-a337-c298d95b7cf3)
